modules/Sources/Generated/Fonts+Generated.swift

```diff
@@ -19,43 +19,32 @@ public typealias SystemFont = FontConvertible.SystemFont
-    public static let regular = FontConvertible(name: "RubikRoman-Regular", family: "Rubik", path: "Rubik-VariableFont_wght.ttf")
-    public static let semiBold = FontConvertible(name: "RubikRoman-SemiBold", family: "Rubik", path: "Rubik-VariableFont_wght.ttf")
-    public static let all: [FontConvertible] = [light, lightItalic, blackItalic, boldItalic, extraBoldItalic, mediumItalic, italic, semiBoldItalic, black, bold, extraBold, medium, regular, semiBold]
+  public enum Inter {
+    public static let black = FontConvertible(name: "Inter-Black", family: "Inter", path: "Inter-Black.otf")
+    public static let blackItalic = FontConvertible(name: "Inter-BlackItalic", family: "Inter", path: "Inter-BlackItalic.otf")
+    public static let bold = FontConvertible(name: "Inter-Bold", family: "Inter", path: "Inter-Bold.otf")
+    public static let boldItalic = FontConvertible(name: "Inter-BoldItalic", family: "Inter", path: "Inter-BoldItalic.otf")
+    public static let extraBold = FontConvertible(name: "Inter-ExtraBold", family: "Inter", path: "Inter-ExtraBold.otf")
+    public static let extraBoldItalic = FontConvertible(name: "Inter-ExtraBoldItalic", family: "Inter", path: "Inter-ExtraBoldItalic.otf")
+    public static let extraLight = FontConvertible(name: "Inter-ExtraLight", family: "Inter", path: "Inter-ExtraLight.otf")
+    public static let extraLightItalic = FontConvertible(name: "Inter-ExtraLightItalic", family: "Inter", path: "Inter-ExtraLightItalic.otf")
+    public static let italic = FontConvertible(name: "Inter-Italic", family: "Inter", path: "Inter-Italic.otf")
+    public static let light = FontConvertible(name: "Inter-Light", family: "Inter", path: "Inter-Light.otf")
+    public static let lightItalic = FontConvertible(name: "Inter-LightItalic", family: "Inter", path: "Inter-LightItalic.otf")
+    public static let medium = FontConvertible(name: "Inter-Medium", family: "Inter", path: "Inter-Medium.otf")
+    public static let mediumItalic = FontConvertible(name: "Inter-MediumItalic", family: "Inter", path: "Inter-MediumItalic.otf")
+    public static let regular = FontConvertible(name: "Inter-Regular", family: "Inter", path: "Inter-Regular.otf")
+    public static let semiBold = FontConvertible(name: "Inter-SemiBold", family: "Inter", path: "Inter-SemiBold.otf")
+    public static let semiBoldItalic = FontConvertible(name: "Inter-SemiBoldItalic", family: "Inter", path: "Inter-SemiBoldItalic.otf")
+    public static let thin = FontConvertible(name: "Inter-Thin", family: "Inter", path: "Inter-Thin.otf")
+    public static let thinItalic = FontConvertible(name: "Inter-ThinItalic", family: "Inter", path: "Inter-ThinItalic.otf")
+    public static let all: [FontConvertible] = [black, blackItalic, bold, boldItalic, extraBold, extraBoldItalic, extraLight, extraLightItalic, italic, light, lightItalic, medium, mediumItalic, regular, semiBold, semiBoldItalic, thin, thinItalic]
```

Comment:
Do we use all of these fonts in the project? E.g., the italic variants might be unnecessary. On the Android side, we use downloadable fonts, so once we approach to use of a new variant, it's downloaded in runtime. So, we don't pack in any unused resources.

Reply:
I simply used the whole font family and did not do any selection. I think in theory we use 2 or 3 out of 19. We don't have any dynamic downloading so it's much easier to use the whole font family than go through the whole process of adding one extra when needed.